doc/src/docs/InputOutputReference/01b-InputOutputReference.md

```diff
@@ -17523,8 +17511,340 @@ This output is the average part-load ratio of the heat pump water heater’s com
+
+#### Field: Availability Schedule Name
+
+This alpha field contains the name of the schedule (ref: Schedule) that denotes whether the heat pump compressor is available to operate during a given time period. A schedule value equal to 0 denotes that the heat pump compressor is off for that time period. A value other than 0 denotes that the heat pump compressor is available to operate during that time period. During times when the heat pump compressor is scheduled off, the heater (element or burner) in the water tank object operates based on its tank set point temperature schedule and the heat pump’s parasitic electric power is also off for that time period. If this field is blank, the schedule has values of 1 for all time periods.
+
+#### Field: Compressor Setpoint Temperature Schedule Name
+
+This alpha field contains the name of the schedule (ref: Schedule) that specifies the set point (or “cut-out”) temperature for the heat pump compressor. Temperature values used in this schedule should be in degrees Celsius. The heat pump compressor cycles off when the tank water reaches this set point temperature. Once the heat pump compressor cycles off, the tank water temperature floats downward until it falls below the set point temperature minus the dead band temperature difference defined below (i.e., the “cut-in” temperature). At this point, the heat pump compressor cycles on and remains on until the heat pump compressor set point temperature is reached.
+
+#### Field: Dead Band Temperature Difference
+
+This numeric field contains the dead band temperature difference in degrees Celsius. The heat pump compressor “cut-in” temperature is defined as the compressor set point temperature defined above minus this dead band temperature difference. The heat pump compressor cycles on when the water temperature in the tank falls below the “cut-in” temperature. The heat pump compressor remains on until the water temperature in the tank rises above the compressor set point (“cut-out”) temperature defined above. The dead band temperature difference must be greater than 0°C and less than or equal to 20°C. If this field is left blank, the default value is 5°C.
+
+In this model, the heat pump water heater’s DX compression system is considered the primary heat source and the water tank’s heater (element or burner) provides supplemental heat as necessary. Therefore, the cut-in temperature for the heat pump compressor (set point minus dead band temperature difference) is usually higher than the set point temperature for the heater (element or burner) in the associated water heater tank object. At times when the water heater tank set point temperature is greater than the cut-in temperature of the heat pump compressor, the heat pump compressor is disabled and the tank’s heater is used to heat the water.
```

Comment:
Last sentence "...heat pump compressor is disabled...", disabled MAY be incorrect as the compressor could turn on if code permitted and the tank water temperature dropped sufficiently. Probably nothing to do here expect to note expected performance and possibly change this wording at a later date.